### ISSUE TITLE
does --full when combined with --enable-gpl automatically add the gpl libraries

### ISSUE THREAD

**Issue by mgood7123** [tanersener/mobile-ffmpeg#394]
does `--full` when combined with `--enable-gpl` automatically add the gpl libraries? for example:
```
./android.sh \
--enable-gpl \
--full \
```
or would i need to do
```
./android.sh \
--speed \
--enable-gpl \
--full \
--enable-rubberband \
--enable-libvidstab \
--enable-x264 \
--enable-x265 \
--enable-xvidcore \
```

**Comment by tanersener** [comment]
No, `--full` does not enable `GPL` licensed libraries even if you add `--enable-gpl` option. You need to enable them explicitly.

This behaviour can be changed. I'll think about it.

**Comment by tanersener** [comment]
Fixed in `development`.

**Comment by mgood7123** [comment]
i just checked out the master branch and the changed didnt work

```
Macs-MacBook-Air:ffmpeg mac$ ./android.sh --speed --enable-gpl --full

Building mobile-ffmpeg library for Android

Architectures: arm-v7a-neon, arm64-v8a, x86, x86-64
Libraries: android-zlib, android-media-codec, fontconfig, freetype, fribidi, gmp, gnutls, lame, libass, libiconv, libtheora, libvorbis, libvpx, libwebp, libxml2, opencore-amr, shine, speex, wavpack, kvazaar, libilbc, opus, snappy, soxr, libaom, chromaprint, twolame, sdl, tesseract, openh264

Building arm-v7a-neon platform on API level 24

^C
Macs-MacBook-Air:ffmpeg mac$ ./android.sh --speed --enable-gpl --full --enable-rubberband --enable-libvidstab --enable-x264 --enable-x265 --enable-xvidcore 

Building mobile-ffmpeg library for Android

Architectures: arm-v7a-neon, arm64-v8a, x86, x86-64
Libraries: android-zlib, android-media-codec, fontconfig, freetype, fribidi, gmp, gnutls, lame, libass, libiconv, libtheora, libvorbis, libvpx, libwebp, libxml2, opencore-amr, shine, speex, wavpack, kvazaar, x264, xvidcore, x265, libvidstab, rubberband, libilbc, opus, snappy, soxr, libaom, chromaprint, twolame, sdl, tesseract, openh264

Building arm-v7a-neon platform on API level 24

^C
Macs-MacBook-Air:ffmpeg mac$ 
```

**Comment by tanersener** [comment]
```
taner@MERA mobile-ffmpeg % git checkout development                
Already on 'development'
Your branch is up to date with 'origin/development'.
taner@MERA mobile-ffmpeg % git pull                                
Already up to date.
Current branch development is up to date.
taner@MERA mobile-ffmpeg % git describe --tags                     
v4.3.2-23-g9df1b5263
taner@MERA mobile-ffmpeg % ./android.sh --speed --enable-gpl --full

Building mobile-ffmpeg library for Android

Architectures: arm-v7a-neon, arm64-v8a, x86, x86-64
Libraries: android-zlib, android-media-codec, fontconfig, freetype, fribidi, gmp, gnutls, lame, libass, libiconv, libtheora, libvorbis, libvpx, libwebp, libxml2, opencore-amr, shine, speex, wavpack, kvazaar, x264, xvidcore, x265, libvidstab, rubberband, libilbc, opus, snappy, soxr, libaom, chromaprint, twolame, sdl, tesseract, openh264, vo-amrwbenc
```

**Comment by tanersener** [comment]
Released in `v4.4`.
